salt/loader.py

```diff
@@ -1062,7 +1064,8 @@ def __init__(self,
         # names of modules that we don't have (errors, __virtual__, etc.)
         self.missing_modules = {}  # mapping of name -> error
         self.loaded_modules = {}  # mapping of module_name -> dict_of_functions
-        self.loaded_files = set()  # TODO: just remove them from file_mapping?
+        self.loaded_files = {}  # TODO: just remove them from file_mapping?
```

Comment:
Jus out of curiosity: is `{}` faster than `set()`? I know there's a performance difference between `[]` and `list()` for example.

However, I looking at that line, I would understand that `self.loaded_files` is a dictionary. So please don't forget to add a comment to specify that is a set.